buildpack.toml

```diff
@@ -5,6 +5,10 @@ api = "0.6"
   id = "paketo-buildpacks/node-engine"
   name = "Paketo Node Engine Buildpack"
 
+  [[buildpack.licenses]]
+    type = "Apache-2.0"
+    uri = "https://github.com/paketo-buildpacks/dotnet-core/blob/main/LICENSE"
```

Comment:
should the `uri` value here contain `nodejs` instead of `dotnet-core` ?

Reply:
yes 😅 . fixed in next commit.